tests/System.IO.Compression.Tests/BrotliPrimitivesTests.cs

```diff
@@ -131,6 +133,34 @@ public void WrongWindowSize(uint windowSize)
 
         [Theory]
         [InlineData(1)]
+        [InlineData(10)]
+        [InlineData(1024)]
+        public void DecompressRandomInvalidData(int totalSize)
+        {
+            Brotli.State state = new Brotli.State();
+            byte[] data = new byte[totalSize];
+            new Random(42).NextBytes(data);
+            Assert.Throws<System.IO.IOException>(delegate { Brotli.Decompress(data, Array.Empty<byte>(), out int consumed, out int written, ref state);  });
```

Comment:
Ah yes. Thanks for clarifying. You could use lambda expressions which are similar (or the same?). But a delegate is fine.
https://github.com/dotnet/corefxlab/blob/master/tests/System.Buffers.Primitives.Tests/CtorArray.cs#L200
```C#
Assert.Throws<System.IO.IOException>(() => Brotli.Decompress(data, Array.Empty<byte>(), out int consumed, out int written, ref state)); 
```